lib/compat/wordpress-6.3/script-loader.php

```diff
@@ -13,3 +13,71 @@
 remove_action( 'wp_body_open', 'wp_global_styles_render_svg_filters' );
 remove_action( 'in_admin_header', 'wp_global_styles_render_svg_filters' );
 
+/**
+ * Collect the block editor assets that need to be loaded into the editor's iframe.
+ *
+ * @since 6.0.0
+ * @access private
+ *
+ * @global string $pagenow The filename of the current screen.
+ *
+ * @return array {
+ *     The block editor assets.
+ *
+ *     @type string|false $styles  String containing the HTML for styles.
+ *     @type string|false $scripts String containing the HTML for scripts.
+ * }
+ */
+function _wp_get_iframed_editor_assets__63() {
```

Comment:
The name of this function wasn't obvious. I was confused by the `__63` and only in retrospect I understand why you're trying to do with this. I'd suggest following the current procedure for naming (use the `gutenberg` prefix instead of `wp`):

```suggestion
function _gutenberg_get_iframed_editor_assets() {
```